.github/workflows/linter.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint Repo
+on:
+  - pull_request
+jobs:
+  build:
+    name: Lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+      - name: Checkout Release from lens
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: git config
+        run: |
+           git config --local user.email "action@github.com"
```

Comment:
You probably don't need these two lines as not committing anything?

Reply:
Fair, will remove